integration/tests/params.go

```diff
@@ -217,3 +226,44 @@ func NewParams(clusterNamePrefix string) *Params {
 	params.GenerateCommands()
 	return &params
 }
+
+// attemptSettingUserID will attempt to fetch the first 2 characters of the userID running the
```

Comment:
small nit after the change 
```suggestion
// attemptSettingUserID will attempt to fetch the first 4 characters of the userID running the
```